pages/mesosphere/dcos/services/elastic/2.8.0-7.3.2/release-notes/index.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: layout.pug
+navigationTitle:
+excerpt: Discover the new features, updates, and known limitations in this release of the Elastic Service
+title: Release Notes
+menuWeight: 10
+model: /mesosphere/dcos/services/elastic/data.yml
+render: mustache
+---
+
+# Release notes for Elastic Service version 2.8.0-7.3.2
```

Comment:
There aren't any breaking changes for customers moving to `7.3.2`? Nice.

But if yes, we should update to `3.0.0-7.3.2`?

Reply:
@phrynchanka let's do this 👆

Reply:
Got it. Thx!